code/game/machinery/suit_storage_unit.dm

```diff
@@ -1084,6 +1084,28 @@
 				suit.item_state = "rig-medical_emt"
 				suit.item_state_slots[slot_r_hand_str] = "medical_voidsuit_emt"
 				suit.item_state_slots[slot_l_hand_str] = "medical_voidsuit_emt"
+		if("Exploration")
+			if(helmet)
+				helmet.name = "exploration voidsuit helmet"
+				helmet.icon_state = "helm_explorer"
+				helmet.item_state = "helm_explorer"

```

Comment:
Unmarked vorestation edits. Can be done in the _vr file too.

Add `/obj/machinery/suit_cycler/proc/apply_paintjob()` to the _vr file, apply the same sanity checks that the original has a the top, and for the switch statement instead just do
```
switch_target(department)
    if(exploration)
        stuff
    if(other thing)
        stuff
    else
        return ..()
```
And it will use your code if they have the department set to one of yours, and then use the original (considered the parent in the case of this weird 'proc defined two places' method) if it is not one.